### ISSUE TITLE
ProCam devices and surface materials employed in each setup

### ISSUE THREAD

**Issue by kylin-leo** [hai-bo-li/SIComp#2]
Interesting work. Would it be possible for you to specify the ProCam devices and surface materials employed in each setup? This information would be valuable for demonstrating the diversity of your dataset.

**Comment by hai-bo-li** [comment]
Thanks for your valuable comment. For the training set, we adopted the Canon 600D camera paired with the EPSON CB‑965 projector.
For the validation set, we used two camera models: Nikon D3200 and Canon 600D; the corresponding projectors are EPSON CB‑965 and TOSHIBA TDP‑T100C. Regarding surface materials in our training dataset, a wide variety are included: real wood board, real plastic board, real fabric, reflective transparent film, coated art paper, offset paper, and standard A2 paper.
